src/ignitus-Shared/ignitus-DesignSystem/ignitus-Molecules/ignitus-Overlay/Components/index.test.tsx

```diff
@@ -40,5 +49,5 @@ describe('<Overlay />', () => {
     expect(wrapper.find('p').length).toBe(2);
   });
 
-  it('should render paragraph with style passed in prop', styleCheckParagraph);
+  it('should render values passed in props properly', propsCheck);
```

Comment:
Place function directly instead of variable. Rest LGTM.

Reply:
Done